gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -4265,7 +4265,7 @@ gnc_plugin_page_register_cmd_view_filter_by (GtkAction* action,
                                   gnc_window_get_gtk_window (GNC_WINDOW (GNC_PLUGIN_PAGE (page)->window)));
 
     /* Translators: The %s is the name of the plugin page */
-    title = g_strdup_printf (_ ("Filter %s by..."),
+    title = g_strdup_printf (_ ("Filter %s by"),
```

Comment:
No, Filter is a command, but it is not immediately executed.

Reply:
This is the dialog's title, so it's not about immediacy of execution. But it would be more correct to use a colon (`:`) than an ellipsis (`…`).